src/theory/arith/nl/iand_utils.h

```diff
@@ -81,10 +84,34 @@ class IAndTable
    *        pass.
    * @return A node that represents the operation, as described above.
    */
-  Node createBitwiseNode(Node x, Node y, uint64_t bvsize, uint64_t granularity);
+  Node createSumNode(Node x, Node y, uint64_t bvsize, uint64_t granularity);
+
+  /** Create a bitwise integer And node for two integers x and y for bits
+   * between hgih and low Example for high = 0, low = 0 (e.g. granularity 1)
+   *    ite(x[0] == 1 & y[0] == 1, #b1, #b0)
+   *
+   *  For higher granularities it makes use of computeAndTable
```

Comment:
Can you use `computeAndTable` for all granularities, including 1? Or is there any disadvantage?

Reply:
Good point! It's actually just an incorrect comment, addressed with c4674e6d2b5717c44ae09a7d317c8c2cc9f2ad3d